### PR TITLE
search for action.yml in sub directories added

### PR DESCRIPTION
**Pull request by hbsan**
Added searching for the action.yml/yaml in any (sub) directories

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -153,12 +153,12 @@ async function findAllActions(
...
     if (content && content.name !== '') {
       core.info(
-        `Found action file in repository: [${repo.name}] with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is [${repo.visibility}]`
+        `Found action file in repository: ${repo.name} with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is ${repo.visibility}`
       )
 
       // Check and exclude the Work in progress actions if its internal
       if (repo.visibility == 'internal') {
-        core.debug(`Get cccess settings for repository [${repo.owner}/${repo.name}] ..............`)
+        core.debug(`Get Access settings for repository ${repo.owner}/${repo.name}..............`)
```
Comment (rajbos) [inline src/main.ts:161]:
```suggestion
        core.debug(`Get access settings for repository ${repo.owner}/${repo.name}..............`)
```

File: src/main.ts [hunk 2]
```diff
@@ -153,12 +153,12 @@ async function findAllActions(
     const content = await getActionFile(client, repo)
     if (content && content.name !== '') {
       core.info(
-        `Found action file in repository: [${repo.name}] with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is [${repo.visibility}]`
+        `Found action file in repository: ${repo.name} with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is ${repo.visibility}`
```
Comment (rajbos) [inline src/main.ts:156]:
```suggestion
        `Found action file in repository: [${repo.name}] with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is [${repo.visibility}]`
```

**Review by rajbos — COMMENTED** [review]
Looks good, some small fixes.

File: dist/main.js [hunk 3]
```diff
@@ -26679,10 +26679,10 @@ function findAllActions(client, repos) {
       const content = yield getActionFile(client, repo);
       if (content && content.name !== "") {
         core.info(
-          `Found action file in repository: [${repo.name}] with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is [${repo.visibility}]`
+          `Found action file in repository: ${repo.name} with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is ${repo.visibility}`
         );
         if (repo.visibility == "internal") {
-          core.debug(`Get cccess settings for repository [${repo.owner}/${repo.name}] ..............`);
+          core.debug(`Get Access settings for repository ${repo.owner}/${repo.name}..............`);
```
Comment (rajbos) [inline dist/main.js:26685]:
```suggestion
          core.debug(`Get access settings for repository ${repo.owner}/${repo.name}..............`);
```

File: dist/main.js [hunk 4]
```diff
@@ -26679,10 +26679,10 @@ function findAllActions(client, repos) {
       const content = yield getActionFile(client, repo);
       if (content && content.name !== "") {
         core.info(
-          `Found action file in repository: [${repo.name}] with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is [${repo.visibility}]`
+          `Found action file in repository: ${repo.name} with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is ${repo.visibility}`
```
Comment (rajbos) [inline dist/main.js:26682]:
```suggestion
          `Found action file in repository: [${repo.name}] with filename [${content.name}] download url [${content.downloadUrl}]. Visibility of repo is [${repo.visibility}]`
```

**Review by rajbos — APPROVED** [review]

**Review by rajbos — APPROVED** [review]

File: dist/main.js (hunk 4 above)
Comment (hbsan) [reply dist/main.js:26682]: This is dist 🐙

**Pull request merged**